config/initializers/00_devise.rb

```diff
@@ -1,3 +1,4 @@
+include EnvironmentHandler
```

Comment:
Let's put it in its own initializer - they run in alphabetical order so I'd call it something like `config/initializers/000_env_helper.rb` or whatever will place it before the Devise initializer.